Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -48,10 +48,13 @@ class AnnotationTrackingState {
 public:
     void startAnnotationTracking(RetainPtr<PDFAnnotation>&&, const WebEventType&, const WebMouseEventButton&);
     void finishAnnotationTracking(const WebEventType&, const WebMouseEventButton&);
-    const PDFAnnotation *trackedAnnotation() const { return m_trackedAnnotation.get(); }
+    const PDFAnnotation* trackedAnnotation() const { return m_trackedAnnotation.get(); }
```

Comment:
For ObjC types we want `const PDFAnnotation *`
https://webkit.org/code-style-guidelines/#pointers-non-cpp

Reply:
Fixed! thanks for catching that